.travis.yml

```diff
@@ -34,4 +34,5 @@ script:
   - echo -en 'travis_fold:end:script.2\\r'
 
   - npm run lint
+  - sleep 120
```

Comment:
lets remove that and add subscription to subgraph to check if it is already synced .
we should add this check for each test and before each of one.